src/machine/machinectl.c

```diff
@@ -61,6 +61,8 @@
 #include "verbs.h"
 #include "web-util.h"
 
+#define dash_if_empty(x) ((x) != NULL && *(x) != 0 ? (x) : "-")
```

Comment:
we have similar calls already in string-util.h in strempty(), strnull(), strna(). I think it would make a lot of sense to add it there too, and also make it a static inline function, to aovid double evaluation. Specifically something like this:

```c
static inline const char* strdash_if_empty(const char *x) {
        return isempty(x)  ? "-" : x;
}
```